examples/web_integration_service/library_api/CMakeLists.txt

```
@@ -0,0 +1,60 @@
+# (c) 2023 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission. Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+cmake_minimum_required(VERSION 3.12)
+project(rticonnextdds-examples-web-integration-service)
+list(APPEND CMAKE_MODULE_PATH
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../resources/cmake/Modules"
+)
+include(ConnextDdsConfigureCmakeUtils)
+connextdds_configure_cmake_utils()
+
+find_package(
+    RTIConnextDDS "7.0.0"
+    REQUIRED
+    COMPONENTS
+        core
+)
+
+add_executable(WebIntegrationServiceLibraryAPICXX11
+    "${CMAKE_CURRENT_SOURCE_DIR}/web_integration_service_library_api.cxx"
+)
+
+set_property(
+    TARGET WebIntegrationServiceLibraryAPICXX11
+    PROPERTY CXX_STANDARD 11
+)
+set_property(
+    TARGET WebIntegrationServiceLibraryAPICXX11
+    PROPERTY CXX_STANDARD_REQUIRED ON
+)
+
+# TODO: Refactor these once https://github.com/rticommunity/rticonnextdds-cmake-utils/issues/75
+# is resolved.
+target_compile_definitions(WebIntegrationServiceLibraryAPICXX11
+    PUBLIC
+        ${CONNEXTDDS_COMPILE_DEFINITIONS})
+
+set_target_properties(WebIntegrationServiceLibraryAPICXX11
+    PROPERTIES
+        DEBUG_POSTFIX "d"
+)
+
+target_include_directories(WebIntegrationServiceLibraryAPICXX11
+    PUBLIC
+        ${CONNEXTDDS_INCLUDE_DIRS}
+)
+
+target_link_directories(WebIntegrationServiceLibraryAPICXX11
+    PUBLIC
+        ${CONNEXTDDS_DIR}/lib/${CONNEXTDDS_ARCH}
+)
+
+target_link_libraries(WebIntegrationServiceLibraryAPICXX11
+    PUBLIC
+        RTIConnextDDS::cpp2_api
+        rtiwebintegrationserviced
+)
```

Apply once https://github.com/rticommunity/rticonnextdds-cmake-utils/pull/76 is merged
```suggestion
        web_integration_service
)

add_executable(WebIntegrationServiceLibraryAPICXX11
    "${CMAKE_CURRENT_SOURCE_DIR}/web_integration_service_library_api.cxx"
)

set_property(
    TARGET WebIntegrationServiceLibraryAPICXX11
    PROPERTY CXX_STANDARD 11
)
set_property(
    TARGET WebIntegrationServiceLibraryAPICXX11
    PROPERTY CXX_STANDARD_REQUIRED ON
)

target_link_libraries(WebIntegrationServiceLibraryAPICXX11
    PUBLIC
        RTIConnextDDS::web_integration_service_cpp2
)
```